brainvisa/toolboxes/data management/processes/database_qc_table.py

```diff
@@ -757,7 +759,7 @@ def save_html(self, filename, context=None):
                         for row, row_id in six.iteritems(rev_row_ids)])
 
     # sort items
-    rows_order = zip(*sorted(zip(list(rev_row_ids.values()), list(range(nrows)))))[1]
+    rows_order = list(zip(*sorted(zip(rev_row_ids.values(), range(nrows)))))[1]
 
```

Comment:
Bug fixed, I think it was not working this python3.

Reply:
probably right ;)